modules/ossm-about-collecting-ossm-data.adoc

```diff
@@ -31,3 +31,12 @@ $ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-mus
 ----
 $ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8:2.4 gather <namespace>
 ----
+
+This creates a local directory that contains the following: 
+
+* The Istio Operator namespace (and its children objects)
+* All control plane namespaces (and their children objects)
+* All namespaces (and their children objects) that belong to any service mesh
+* All Istio custom resource definitions (CRD)
+* All Istio CRD objects (VirtualServices in a given namespace, etc)
```

Comment:
```suggestion
* All Istio CRD objects, such as VirtualServices in a given namespace
```
Per IBM Style, the term _etc_ is a do-not-use term. Here's the guidance: "Do not use. Use _and so on_ when you list a clear sequence of elements, such as _1, 2, 3, and so on_ or _Monday, Tuesday, Wednesday, and so on_. When you list items that do not form a clear sequence, use appropriate descriptive wording, such as _device drivers, firmware, and other code_."